modules/manifold/src/components/ui/segment-filters-control/segment-filters-control.js

```diff
@@ -0,0 +1,156 @@
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import clone from 'lodash.clone';
+import {generateRandomId} from '@mlvis/mlvis-common/utils';
+import Plus from 'baseui/icon/plus';
+
+import {THEME_COLOR, STATE_DATA_TYPES, FIELD} from '../../../constants';
+import {dotSet} from '../../../utils';
+import SegmentPanelList from './segment-panel-list';
+
+const Container = styled.div`
+  position: relative;
+`;
+
+const Header = styled.div`
+  position: absolute;
+  right: 0;
+  top: -28px;
+`;
+
+const Footer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-bottom: 8px;
+`;
+
+const ConfirmButton = styled.div`
+  cursor: ${props => (props.disabled ? 'not-allowed' : 'pointer')};
+  color: ${props => (props.disabled ? '#aaa' : THEME_COLOR)};
+`;
+
+const IconButton = styled.div`
+  height: 24px;
+  padding-right: 8px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  margin: 1px;
+  transition: 0.2s;
+`;
+
+const _getSegmentFiltersStateFromProps = segmentFilters =>
+  segmentFilters.map((filters, segmentId) => ({
+    // _key is only use as keys to segment panels
+    _key: generateRandomId(),
+    _content: filters,
```

Comment:
Why use underscore for these two? e.g. we are accessing `_content` in line 97, seems it is not a private field.

From the usage below, it seems `_content` is just `filters`, let's just use `filters` which is more explicit.

Reply:
Done.